policies/coding-style.md

```diff
@@ -538,8 +538,9 @@ The [GNU cpp manual][8] deals with macros exhaustively.
 
 ## Chapter 10: Allocating memory
 
-OpenSSL provides the following general purpose memory allocators:
-`OPENSSL_malloc()`, `OPENSSL_realloc()`, `OPENSSL_strdup()` and `OPENSSL_free()`.
+OpenSSL provides many general purpose memory utilities, including, but
+not limited to: `OPENSSL_malloc()`, `OPENSSL_zalloc()`, `OPENSSL_realloc()`,
+`OPENSSL_memdup()`, `OPENSSL_strdup()` and `OPENSSL_free()`.
```

Comment:
I added my favorite allocation utilities; there are more, so indicated that.

Reply:
add comma before and?

Reply:
Ack! I forgot the Oxford comma!